mod/wikirate/set/right/activity.rb

```diff
@@ -36,4 +36,13 @@ def activity_item item
       </div>
     }.html_safe
   end
+
+  view :open do |args|
+    if !card.accountable? && !card.account
```

Comment:
I think we only want `!card.account`.

Pretty soon all company cards will be accountable, but we still won't want to show activity unless they actually have an account setup.